prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -5,15 +5,16 @@
 
 import tenacity
 from crewai import Agent, Crew, Process, Task
-from langchain.utilities.tavily_search import TavilySearchAPIWrapper
 from langchain_community.tools.tavily_search import TavilySearchResults
+from langchain_community.utilities.tavily_search import TavilySearchAPIWrapper
 from langchain_core.callbacks import (
     AsyncCallbackManagerForToolRun,
     CallbackManagerForToolRun,
 )
 from langchain_core.language_models import BaseChatModel
 from langchain_core.pydantic_v1 import SecretStr
 from langchain_openai import ChatOpenAI
+from loky import get_reusable_executor
```

Comment:
Use this instead of the previously used function from PMAT https://github.com/joblib/loky

The previous implementation works just fine locally and initially worked in Kubernetes, too. I don't know what changed, but this seems to fix it.

They are more fancy:

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/1238423c-21a1-4709-9d4d-e18bc9e8e83f)